pkg/asset/agent/installconfig.go

```diff
@@ -216,13 +221,9 @@ func (a *OptionalInstallConfig) validateControlPlaneConfiguration(installConfig
 	var fieldPath *field.Path
 
 	if installConfig.ControlPlane != nil {
-		if *installConfig.ControlPlane.Replicas < 1 || *installConfig.ControlPlane.Replicas > 5 || (installConfig.Arbiter == nil && *installConfig.ControlPlane.Replicas == 2) {
+		if *installConfig.ControlPlane.Replicas < 1 || *installConfig.ControlPlane.Replicas > 5 || *installConfig.ControlPlane.Replicas == 2 {
 			fieldPath = field.NewPath("ControlPlane", "Replicas")
-			supportedControlPlaneRange := "to 5, 4, 3, or 1"
-			if installConfig.EnabledFeatureGates().Enabled(features.FeatureGateHighlyAvailableArbiter) {
```

Comment:
Could you elaborate why this was removed?

Reply:
fixed